api/views.py

```diff
@@ -127,7 +127,8 @@ class Meta:
         }
 
 
-class VolunteersViewSet(mixins.ListModelMixin, viewsets.GenericViewSet):
+class VolunteersViewSet(mixins.ListModelMixin, mixins.DestroyModelMixin, mixins.RetrieveModelMixin,
+                        viewsets.GenericViewSet):
     queryset = Volunteer.objects.all().order_by('-created_date').annotate(time_volunteered=Count('helprequest'))
```

Comment:
There's a method in the manages to add times volunteered (all_with_helprequests_count).

Save the annotation.